fudge-ratio/src/main/webapp/calculator.html

```diff
@@ -5,7 +5,7 @@
         <link rel="stylesheet" href="calculator_style.css" />
         <title>Fudge Ratio</title>
     </head>
-    <body onload="loadActivities()">
```

Comment:
Onload is not a good idea here:
* This is a HTML file, so the contents should only be semantic, html should be agnostic to the style of the page and to the code running, so using a javascript function here seems wrong
* This assumes that the function ``loadActivities`` has already been declared, but this may not be the case, 
in fact I am astonished that this work, because we are using the defer attribute in the script, so the browser first read the html and parse it and just then start working on the javascript

Reply:
Thanks, this is really interesting, I didn’t realize this when I went through the code .